opennms-model/src/main/java/org/opennms/netmgt/model/OnmsNode.java

```diff
@@ -1014,6 +1014,13 @@ public void removeRequisitionedCategory(final String category) {
         return m_metaData;
     }
 
+    public Optional<OnmsMetaData> getMetaDataForContextAndKey(final String context, final String key) {
```

Comment:
Maybe call it `findMetaDataByContextAndKey` as this can become quite expensive

Reply:
Ok